docs/credentials.md

```diff
@@ -150,11 +150,19 @@ Define the value as a reference to the corresponding key in the credentials file
 
 -  `_CREDS` is an environment constant pointing to the `.credentials` file
 -  `my_data_key` is a key in the the `.credentials` file or vault that contains the value to be used in a test step
+   - for File Storage, ensure your key contains the vendor prefix, i.e. `vendor/my_data_key`
+   - for Vault Storage, ensure your key contains your secret basepath + `/mftf/` + your vendor prefix + `/` and finally your key, i.e. `secret/mftf/vendor/my_data_key`
+
+For example, to reference secret data in the [`fillField`][] action, use the `userInput` attribute using a typical File Storage:
+
+```xml
+<fillField stepKey="FillApiToken" selector=".api-token" userInput="{{_CREDS.vendor/my_data_key}}" />
+```
 
-For example, reference secret data in the [`fillField`][] action with the `userInput` attribute.
+To do the same using Vault Storage:
 
 ```xml
-<fillField stepKey="FillApiToken" selector=".api-token" userInput="{{_CREDS.my_data_key}}" />
+<fillField stepKey="FillApiToken" selector=".api-token" userInput="{{_CREDS.secret/mftf/vendor/my_data_key}}" />
```

Comment:
Thanks @KevinBKozan ! I actually do not have Vault set up locally, so I was completely shooting in the dark with the Vault portion of the docs.

Do you have suggestions on how to modify this section? Ideally, I would imagine the best developer experience here would be to use the same token to access the secret regardless of the underlying storage leveraged. If that is the case, then I can definitely simplify the wording in the docs here to make that clear! Just let me know.

Reply:
Gotcha. I think some background info is kind of necessary to be able to make good changes to the docs, but the tl;dr is `Vault integration with MFTF is about 50% done in the whole picture (framework integration is complete, build/EQP integration is not)`.

The primary purpose of the vault integration is actually to connect to the (as of yet, non-existant) MFTF vault credential store. This is the thing that builds are going to be using to run tests, so it becomes the "canonical" list of credentials that if you submit an extension, you can submit creds to your sandbox or testing environment.

So with that said, for local test running the prefered way would actually be for you to just define creds in `.credentials` as you need them. If you're running a test that requires `_CREDS.some_api_key` it'd be then on your to define it in `.credentials` since you would conceivably know the value (if you have access to it).

You can *still* use vault to resolve these `_CREDS` references though. So say a test is using `_CREDS.some_api_key`, the user flow would have to be
* `brew install vault`
* `vault login <username/token/whatever EQP decides to use>`
** vault is now authenticated on the machine, MFTF will use this auth to access creds
** you don't need to tell vault where the server is AFAIK, it will know by your token to map it to the right server
* define details in `.env`:
** `CREDENTIAL_VAULT_ADDRESS=<whatever it ends up being>`
** `CREDENTIAL_VAULT_SECRET_BASE_PATH=<probably just "secret">`
** MFTF now knows what the vault address is, what the base path is, and has auth to access it

Reply:
So I guess the followup question is, what are *you* trying to use vault for, connecting to that central service or a local instance?

Reply:
Oh, I'm using File Storage (`.credentials` file) locally so the updated File Storage docs is all I would add. I assumed Vault Storage  was ready; if  it's not, perhaps best to leave out any details for now. What do you think?

Reply:
You're honestly probably right. We can make reasonable assumptions about how it all ends up being implemented outside MFTF (as a framework), but we can leave out details for now. 👍

Reply:
Cool, I've updated the docs to remove the Vault details.